docs/index.md

```diff
@@ -34,11 +34,6 @@ id: home
         JSX. Input data that is passed into the component can be accessed by
         `render()` via `this.props`.
       </p>
-      <p>
-        <strong>JSX is optional and not required to use React.</strong> Try
```

Comment:
Can we keep this and add a link to the Babel repl instead?

Reply:
Agreed with keeping the "JSX is optional..." bit. Don't feel strongly either way about linking to the Babel REPL.

Reply:
I think it would be nice to link to something - I find it a bit of a leap to send them to the Babel repl; it looks like the default doesn't have any React or JSX. https://goo.gl/d1eRN4

Either way I'll put back the 'JSX is optional' comment.

Reply:
We could link to

http://babeljs.io/repl#?babili=false&browsers=&build=&builtIns=false&code_lz=MYGwhgzhAEASCmIQHsCy8pgOb2vAHgC7wB2AJjAErxjCEB0AwsgLYAOyJph0A3gFABIAE6ky8YQAoAlHyEj4hAK7CS0ADxkAlgDcAfAiTI-hABZaI9NsORtLJMC3gBfdQHpt-gNxDn_P_zUtIQAIgDyqPSi5BKS6oYo6Jg40A5OALwARCHwOlokmdBuegA00CzISiSEAHLI4tJeQA&debug=false&circleciRepo=&evaluate=false&lineWrap=false&presets=react&prettier=false&targets=&version=6.26.0

Reply:
:grin: I do kind of like this notion now that I think about it more

Reply:
**JSX is optional and not required to use React.** Try the [Babel REPL](http://babeljs.io/repl#?babili=false&browsers=&build=&builtIns=false&code_lz=MYGwhgzhAEASCmIQHsCy8pgOb2vAHgC7wB2AJjAErxjCEB0AwsgLYAOyJph0A3gFABIAE6ky8YQAoAlHyEj4hAK7CS0ADxkAlgDcAfAiTI-hABZaI9NsORtLJMC3gBfdQHpt-gNxDn_P_zUtIQAIgDyqPSi5BKS6oYo6Jg40A5OALwARCHwOlokmdBuegA00CzISiSEAHLI4tJeQA&debug=false&circleciRepo=&evaluate=false&lineWrap=false&presets=react&prettier=true&targets=&version=6.26.0) to see the raw JavaScript code produced by the JSX compilation step.